DTIPrep/DTI/DTI.pm

```diff
@@ -404,12 +404,12 @@ sub determinePreprocOutputs {
 =head3 determinePostprocOutputs($QCoutdir, $dti_file, $anat, $protXMLrefs)
 
 Function that will determine post processing output names (for either DTIPrep
-or mincdiffusion postprocessing) and append them to C<$DTIrefs>.
+or C<mincdiffusion> postprocessing) and append them to C<$DTIrefs>.
 
 INPUTS:
   - $QCoutdir   : directory that will contain output files
   - $dti_file   : raw DWI file to be processed
-  - $anat       : T1 image to be used for mincdiffusion postprocessing
+  - $anat       : T1 image to be used for C<mincdiffusion> postprocessing
```

Comment:
on line 778, there is a `nrrd2minc` so maybe you cna put it in C<> there as well?

Reply:
line 778 is a simple comment for the code that would not go into the documentation. We should probably only format what is between `=pod` and `=cut` and not comments to the code. What do you think?

Reply:
Okay!